src/components/Form/FormFields/TextAreaFormField.tsx

```diff
@@ -44,6 +44,11 @@ const TextAreaFormField = forwardRef(
           )}
           onFocus={props.onFocus}
           onBlur={props.onBlur}
+          onKeyDown={(e) => {
+            if (e.key === "Enter" && !e.shiftKey && !e.metaKey) {
+              e.stopPropagation();
+            }
+          }}
```

Comment:
why do we need these hacks? can't we stick with html standards?

Reply:
The condition is added to handle different behaviors for components like DiscussNote (where Shift + Enter should send a message) and forms like Add Patient (where just Enter should submit the form). By stopping the propagation only when Enter is pressed, it ensures both use cases work as intended without interference.

Reply:
are you sure that textarea element triggers a "submit" event when pressing enter in the first place? i don't think the issue is with TextAreaFormField itself in the first place.

Reply:
The `Add Patient` submit action gets trigger on pressing `enter` key

Reply:
https://github.com/ohcnetwork/care_fe/blob/37dce64eb66d65906f2396742f1f6ce2d7ac6b83/src/components/Form/Form.tsx#L102

Reply:
and is textarea form field responsible for it?

do try replicating this with simple html textarea and form component. pressing enter on textarea does not trigger form submit by itself.

Reply:
isn't this the root issue?

https://github.com/ohcnetwork/care_fe/blob/37dce64eb66d65906f2396742f1f6ce2d7ac6b83/src/components/Form/Form.tsx#L101-L105